test/data/mock.php

```diff
@@ -233,6 +233,19 @@ protected function errorWithScript( $req ) {
 		}
 	}
 
+	protected function errorWithJSONPAndResponseBodyScript( $req ) {
```

Comment:
This function shouldn't be needed since the above `errorWithScript` already does the same, doesn't it?

If you had to add something here, `mock.php` is a PHP file but we also have Karma-based tests with a Node.js server which is what gets run on Travis, for example (or if you run `npm test` locally). That server is defined in `test/middleware-mockserver.js` and it needs to mirror all the definitions here. As the tests already used the `mock.php` name in the URLs, the Node.js server also responds on them.